src/types.ts

```diff
@@ -26,5 +32,5 @@ export type SlackAPIMethodArgs = BaseMethodArgs & {
 };
 
 export type SlackAPIMethod = {
-  (args: SlackAPIMethodArgs): Promise<BaseResponse>;
+  (args?: SlackAPIMethodArgs): Promise<BaseResponse>;
```

Comment:
Made this optional, and default to an empty object in apiCall() so that it's not required for methods that have no required arguments.